cmd/fyne/commands/command.go

```diff
@@ -49,3 +49,10 @@ func NewPackager() Command {
 func NewReleaser() Command {
 	return &commands.Releaser{}
 }
+
+// NewServer returns a command that can serve wasm application over http.
+//
+// Deprecated: A better version will be exposed in the future.
+func NewServer() Command {
+	return &commands.Server{}
+}
```

Comment:
Why introduce a new API that is already deprecated? We created the internal commands package to avoid this. From what I can see, this is not used anywhere so it should be able to safely be removed.

Reply:
I kind of wanted to keep things consistent with all the other command. If this is not necessary, I can remove all of that easily.

Reply:
Yeah well caught @Jacalz - we will be removing these around 3.0 which may be after 2.2.
They are there because it's technically public API that apps could have utilised, but adding new ones in the same place should not be needed/

Reply:
Yeah. It's better to just remove this. As Andy said, it's only there with the other commands because it has to be like that.